lib/openscap_parser/rule.rb

```diff
@@ -83,6 +83,10 @@ def parent_type
       end
     end
 
+    def rule_values
+      @rule_values ||= parsed_xml.xpath("check/check-export").map { |r| r.at_xpath('@value-id')&.text }
+    end
```

Comment:
This gets a map of the ids for the values that the rule is associated with. During the import, we'll need to make sure we're associated the correct `rules` with the correct `values` because the `value` that the `rule` is referencing can be different depending on what `profile` the `rule` belongs to.

Reply:
I see, okay. Please don't work on Thanksgiving :laughing: